File: library/src/main/java/com/example/library/dto/response/MemberResponse.java
package com.example.library.dto.response;

import java.time.LocalDateTime;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class MemberResponse {

    private Integer id;

    private String cardNumber;

    private String name;

    private Integer createdById;

    private String createdByUsername;
    
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt; 

}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/MemberResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDateTime

data class MemberResponse(
    val id : Int,
    val cardNumber : String,
    val name : String,
    val createdByUsername : String,
    val createdById : Int,
    val updatedAt: LocalDateTime,
    val deleteAt: LocalDateTime?
)
